cmd/apiserver/main.go

```diff
@@ -46,6 +47,8 @@ func main() {
 	flag.StringVar(&s.restCfg.LeaderConfig.LockName, "lock-name", "apiserver-lock", "the lease lock resource name")
 	flag.DurationVar(&s.restCfg.LeaderConfig.Duration, "duration", time.Second*5, "the lease lock resource name")
 	flag.DurationVar(&s.restCfg.AddonCacheTime, "addon-cache-duration", time.Minute*10, "how long between two addon cache operation")
+	flag.StringVar(&usecase.DexIssuerURL, "dex-issuer-url", "https://kubevela-dex-issuer.com", "the issuer url of dex")
```

Comment:
This URL be called from vela-apiserver or browser? 

For browser,  we will proxy by VelaUX. such as "http://velaux.public/dex"

For vela-apiserver, it's better that directly through the service name.